drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -340,6 +340,49 @@ def data_collection_features
     end.compact
   end
 
+  def occurrence_point_forms
+    project.occurrence_point_form_instances.map do |instance|
+      matches = instance.project_and_enrollment_match(project: project, enrollment: self).present?
+      has_any_data = false
+
+      unless matches # Only do this computation if the instance doesn't match this enrollment
+        instance.definition.walk_definition_nodes(as_open_struct: true) do |item|
+          next unless item.mapping.present?
+
+          record_type = item.mapping&.record_type
+          field_name = item.mapping&.field_name&.underscore
+          custom_field_key = item.mapping&.custom_field_key
+
+          next unless record_type == 'ENROLLMENT' || custom_field_key
+
+          if record_type && field_name
+            next unless respond_to?(field_name)
+
+            has_any_data = send(field_name).present?
+          elsif custom_field_key
+            # For simplicity, for now, just look for CDEDs where the owner is an Enrollment or a Client
```

Comment:
Add comment to make more clear - this has only been tested to work in specific cases. Drop the client thing for now